README.md

```diff
@@ -47,6 +47,9 @@ api.remove(object)
 api.remove([object1, id2])
 api.removeAll()
 api.removeAll(filterFunction)
+api.on(eventName, eventHandlerFunction)
+api.one(eventName, eventHandlerFunction)
+api.off(eventName, eventHandlerFunction)
 ```
```

Comment:
again, we can do this after merge (let's just create an issue), but maybe we could also mention all available events like this? Thoughts?

```js
// events
hoodie.store.on('add', function(object, options) {})
hoodie.store.on('update', function(object, options) {})
hoodie.store.on('remove', function(object, options) {})
hoodie.store.on('change', function(eventName, object, options) {})
hoodie.store.one(event, eventHandlerFunction)
hoodie.store.off(event, eventHandlerFunction)
```